internal/pkg/workspace/workspace.go

```diff
@@ -442,12 +443,14 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("read directory: %w", err)
 	}
-	var directories []string
+	var dockerfiles = make([]string, 0)
```

Comment:
why do we change to use `make([]string, 0)` 🤔

Reply:
Consistency with previous behavior which returned an empty slice, not the nil slice. This can be changed easily